package.json

```diff
@@ -273,7 +274,7 @@
     "source-map-loader": "^0.1.6",
     "style-loader": "^0.23.1",
     "styled-components": "^4.2.1",
-    "superagent": "^3.8.3",
+    "superagent": "5.2.2",
```

Comment:
We should probably update `superagent` for all subpackages as well to avoid unwanted side effects.

Reply:
👍